deploy/helm-chart/values.yaml

```diff
@@ -0,0 +1,59 @@
+gateway:
+  nginxGateway:
+    ## A GatewayClass resource with the name equal to the class must be deployed. Otherwise, NKG will fail to start.
```

Comment:
We should avoid using acronyms in user-facing docs for our product name

Reply:
Yes, users don't know our products via acronyms like we do, so docs team has previously told us to avoid using them in public-facing docs.